src/features/home/courses/lib/formatCoursesData.ts

```diff
@@ -0,0 +1,19 @@
+import dayjs from 'dayjs';
+import { Course } from '@/app/types';
+
+const now = dayjs();
+const courseExpirationDate = now.subtract(3, 'week');
+
+export const formatCoursesData = (courses: Course[]) => {
+  return courses
+    ?.filter((course) => {
+      const startDate = dayjs(course.startDate);
+      return startDate > courseExpirationDate;
```

Comment:
There's an `isAfter` query in the date-library.
You can add it in this file
![image](https://github.com/rolling-scopes/site/assets/79564962/18e320d7-9070-4023-8c67-7a636a13f623)